errbot/bootstrap.py

```diff
@@ -63,6 +63,8 @@ def bot_config_defaults(config):
         config.TEXT_DEMO_MODE = True
     if not hasattr(config, 'BOT_ADMINS'):
         raise ValueError('BOT_ADMINS missing from config.py.')
+    if not hasattr(config, 'BOT_ADMINS_MUTE_NOTIFICATIONS'):
```

Comment:
I can change the behaviour to make it the notification list, as opposed to an exclusion list of admins.